api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,394 @@ paths:
           description: Partition id
           type: string
           required: true
-        - name: X-Flow-Id
-          in: header
+        - $ref: '#/parameters/XFlowId'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Partition'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /subscriptions:
+    put:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a subscription for EventTypes. The subscription is needed to be able to
+        consume events from EventTypes in a high level way when Nakadi stores the offsets and manages the
+        rebalancing of consuming clients.
+        The subscription is identified by its key parameters (owning_application, event_types, consumer_group). If
+        this endpoint is invoked several times with the same key subscription properties in body (order of even_types is
+        not important) - the subscription will be created only once and for all other calls it will just return
+        the subscription that was already created.
+      parameters:
+        - name: subscription
+          in: body
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: true
+      responses:
```

Comment:
We removed `Content-Location` in favor of `Location` https://github.com/zalando/nakadi/pull/353/commits/26a1d24ac819d76b2e2fb2439d01b7d93a6b64a4

But we are sure if we should always reply with `201` and remove `200` from the API since there would be no means for the client to figure out if the subscription was just created of if already existed. Maybe that's not an issue at all and we should only remove 200.

200 is an issue because, as you mentioned, it doesn't define what `Location` means.